.github/actions/install-dashboards/action.yml

```diff
@@ -33,28 +34,39 @@ runs:
         ref: 'main'
         fetch-depth: 0
 
-    - run: mkdir -p plugins
-      working-directory: OpenSearch-Dashboards
-      shell: bash
-
     - uses: actions/checkout@v2
       with:
         path: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
 
-    - id: osd-version
-      continue-on-error: true
+    - id: osd-version-linux
+      if: ${{ runner.os == 'Linux' }}
       run: |
-        echo "::set-output name=osd-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-4)"
+        echo "::set-output name=osd-version-linux::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-4)"
         echo "::set-output name=osd-x-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-3)"
       working-directory: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
       shell: bash
 
-    - id: branch-switch-if-possible
+    - id: osd-version-windows
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        echo "::set-output name=osd-version-linux::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-4)"
```

Comment:
Did you mean `osd-version-windows` here? Also, we can combine it with `osd-version-linux` to make `osd-version`